L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker.cc

```diff
@@ -677,6 +709,13 @@ void L1TrackObjectNtupleMaker::beginJob() {
   m_2ltrkjetExt_nTightDisplaced=new std::vector<int>;
   m_2ltrkjetExt_ntdtrk=new std::vector<int>;
 
+  m_trkjetemExt_pt = new std::vector<float>;	
+  m_trkjetemExt_phi = new std::vector<float>;	
+  m_trkjetemExt_eta = new std::vector<float>;	
+  m_trkjetemExt_z = new std::vector<float>;	
```

Comment:
Seems that you introduce this variable but then never write the branch, Is it just a negligence?

Reply:
The branch is now written so we can study the z position of our jets.